app/uk/gov/hmrc/helptosavefrontend/views/register/account_created.scala.html

```diff
@@ -46,69 +47,85 @@
 }
 
 @layout(title) {
-
-@if(hasErrors) {
-@errorSummary(form.errors)
-}
-
-  <div class="govuk-box-highlight">
-    <h1 class="govuk-heading--xl">@title</h1>
-    <span class="heading-large">
+        @if(form.errors.nonEmpty) {
+            @viewHelpers.govukErrorSummary(ErrorSummary(errorList =
+                form.errors.map { error =>
+                ErrorLink(
+                href = Some(s"#${error.key}"),
+                content = Text(messages("hts.payInNow.error"))
+                )
+                },
+                title = Text(messages("hts.global.error-summary.title"))
+            ))
+    }
+  <div class="govuk-panel govuk-panel--confirmation">
```

Comment:
this could be using GovukPanel with something like 
```
@viewHelpers.govukPanel(Panel(
    title = Text(title),
    content = HtmlContent(messages("hts.account-created.title.p1"))
  ))
```